pkg/types/spec/validations.go

```diff
@@ -870,7 +870,7 @@ func GetTFServingExportFromLocalPath(path string) (string, error) {
 	if err := files.CheckDir(path); err != nil {
 		return "", err
 	}
-	paths, err := files.ListDirRecursive(path, false, files.IgnoreHiddenFiles, files.IgnoreHiddenFolders)
+	paths, err := files.ListDirRecursive(path, false, files.IgnoreHiddenFilesExceptDotEnv, files.IgnoreHiddenFolders)
```

Comment:
Is it necessary to include `.env` files here? It probably doesn't matter either way, so it's ok to leave it as is I think (the alternative would be to bring back `files.IgnoreHiddenFiles()`)

Reply:
Modified it to not rely on `IgnoreHiddenFiles` but by adding the `.env` path to the list of paths if the file exists.